realm/realm-library/src/androidTest/java/io/realm/ColumnIndicesTests.java

```diff
@@ -81,18 +82,20 @@ public void copyDeeply() {
         final long schemaVersion = 100;
 
         final ColumnIndices columnIndices = create(schemaVersion);
-        final ColumnIndices deepCopy = columnIndices.clone();
+        final ColumnIndices deepCopy = new ColumnIndices(columnIndices, true);
```

Comment:
That could work, but then you don't have the `copy` method.  You need copy for `copyFrom`, and you need `copyFrom` to make @zaki50 's clever fast update scheme to work.